kratos/processes/check_same_modelpart_using_skin_distance_process.cpp

```diff
@@ -0,0 +1,172 @@
+    const double error = IndexPartition<std::size_t>(r_model_part_1.NumberOfElements()).for_each<SumReduction<double>>([&](std::size_t i) {
+        auto it_elem_1 = it_elem_begin_1 + i;
+        auto it_elem_2 = it_elem_begin_2 + i;
+        const Vector& r_elem_dist_1 = it_elem_1->GetValue(r_elem_dist_var);
+        const Vector& r_elem_dist_2 = it_elem_2->GetValue(r_elem_dist_var);
+        return norm_2(r_elem_dist_1 - r_elem_dist_2);
+    });
+
+    if (error > tolerance) {
+        KRATOS_ERROR << "The distance between the two model parts is " << error << " and the tolerance is " << tolerance << std::endl;
+    } else {
+        KRATOS_INFO("CheckSameModelPartUsingSkinDistanceProcess") << "The distance between the two model parts is " << error << " and the tolerance is " << tolerance << ". It would be assumed it is the same model part." << std::endl;
+    }
+
+    // TODO: Add MPI version
+
+    // Cleaning up created model parts
+    mrModel.DeleteModelPart("BACKGROUND_MESH_1");
+    mrModel.DeleteModelPart("BACKGROUND_MESH_2");
+
+    KRATOS_CATCH("")
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+template<std::size_t TDim>
+const Parameters CheckSameModelPartUsingSkinDistanceProcess<TDim>::GetDefaultParameters() const
+{
+    KRATOS_TRY
+
+    const Parameters default_parameters = Parameters(R"(
+    {
+        //"model_part_1_name"                   : "PLEASE_SPECIFY_MODEL_PART_1_NAME", // The background mesh is generated automatically
+        "skin_model_part_1_name"              : "PLEASE_SPECIFY_SKIN_MODEL_PART_2_NAME",
+        //"model_part_2_name"                   : "PLEASE_SPECIFY_MODEL_PART_2_NAME", // The background mesh is generated automatically
+        "skin_model_part_2_name"              : "PLEASE_SPECIFY_SKIN_MODEL_PART_2_NAME",
+        "tolerance"                           : 1.0e-3,
+        //"continuous_distance"               : false, // TODO: Add continuous version if needed in the future
+        "bounding_box_scale_factor"           : 1.5,
+        "number_of_divisions_background_mesh" : 30,
+        "discontinuous_distance_settings": {
+            "elemental_distances_variable"                          : "ELEMENTAL_DISTANCES",
+            "elemental_edge_distances_variable"                     : "ELEMENTAL_EDGE_DISTANCES",
+            "elemental_edge_distances_extrapolated_variable"        : "ELEMENTAL_EDGE_DISTANCES_EXTRAPOLATED",
+            "embedded_velocity_variable"                            : "EMBEDDED_VELOCITY",
+            "calculate_elemental_edge_distances"                    : false,
+            "calculate_elemental_edge_distances_extrapolated"       : false,
+            "use_positive_epsilon_for_zero_values"                  : true
+        }
```

Comment:
If we eventually change the discontinuous distance settings we'll require to update this manually. To avoid so, I'd call the discontinuous distance `GetDefaultParameters` to get these and place them in the `discontinuous_distance_settings`.
```suggestion
        "discontinuous_distance_settings": {
        }
```